python/labbox_ephys/workspace/workspace.py

```diff
@@ -6,9 +6,9 @@ class Workspace:
     def __init__(self, *, feed: kp.Feed, workspace_name: str) -> None:
```

Comment:
The Workspace Python class represents a labbox workspace. It is defined by a feed and a workspace name. The corresponding subfeed is: `sf = feed.get_subfeed({'workspaceName': workspace_name})`